src/include/platform/ConnectivityManager.h

```diff
@@ -443,6 +439,24 @@ inline CHIP_ERROR ConnectivityManager::GetAndLogWiFiStatsCounters()
 }
 
 #if CHIP_DEVICE_CONFIG_ENABLE_WIFIPAF
+inline CHIP_ERROR ConnectivityManager::SetWiFiPAFAdvertisingEnabled(bool val)
+{
+    if (val)
+    {
+        return WiFiPAFPublish(mPafAdverParam);
+    }
+    return WiFiPAFCancelPublish(mPafAdverParam.publish_id);
+}
+
+inline CHIP_ERROR ConnectivityManager::SetWiFiPAFPublishParam(WiFiPAFAdvertiseParam & args)
```

Comment:
should the set input be const ?

Reply:
otherwise given that this is a uniqe_ptr already, does it make sense to std::move that data instead?

Reply:
I will add "const", or will use the different way, if using "ReadOnlyBuffer"